<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1629875065891' stop='1629875106627'>
    <name>verify generate-download-share funtion verify project tab</name>
    <title>verify generate-download-share funtion verify project tab</title>
    <test-cases>
        <test-case start='1629875065891' status='passed' stop='1629875083839'>
            <name>Generate-new-dataset.</name>
            <title>Generate-new-dataset.</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
        </test-case>
        <test-case start='1629875083839' status='passed' stop='1629875094330'>
            <name>Download project.</name>
            <title>Download project.</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
        </test-case>
        <test-case start='1629875094330' status='failed' stop='1629875101533'>
            <name>Share project.</name>
            <title>Share project.</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='bdae653a-01d5-46c8-ab32-6829ba1c588e-attachment.png' type='image/png' size='76886'/>
            </attachments>
            <failure>
                <message>Expected '[object Promise]' to equal 'folder-open'.</message>
                <stack-trace>Error: Failed expectation
                    at Object.&lt;anonymous> (E:\vmware\Loop\GitHub\e2e\features\download-share-projects-spec.ts:51:71)
                    at Generator.next (&lt;anonymous>)
                    at fulfilled (E:\vmware\Loop\GitHub\e2e\node_modules\tslib\tslib.js:112:62)
                    at processTicksAndRejections (node:internal/process/task_queues:96:5)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1629875101533' status='failed' stop='1629875104055'>
            <name>Share numeric project.</name>
            <title>Share numeric project.</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='c3f85800-b107-479e-81bd-43fdb7ea6189-attachment.png' type='image/png' size='75640'/>
            </attachments>
            <failure>
                <message>Expected '[object Promise]' to equal 'folder-open'.</message>
                <stack-trace>Error: Failed expectation
                    at Object.&lt;anonymous> (E:\vmware\Loop\GitHub\e2e\features\download-share-projects-spec.ts:58:71)
                    at Generator.next (&lt;anonymous>)
                    at fulfilled (E:\vmware\Loop\GitHub\e2e\node_modules\tslib\tslib.js:112:62)
                    at processTicksAndRejections (node:internal/process/task_queues:96:5)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1629875104055' status='failed' stop='1629875106625'>
            <name>Share log project.</name>
            <title>Share log project.</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='45e55bf0-7ba3-4852-b4eb-3e6821fa4428-attachment.png' type='image/png' size='75627'/>
            </attachments>
            <failure>
                <message>Expected '[object Promise]' to equal 'folder-open'.</message>
                <stack-trace>Error: Failed expectation
                    at Object.&lt;anonymous> (E:\vmware\Loop\GitHub\e2e\features\download-share-projects-spec.ts:66:71)
                    at Generator.next (&lt;anonymous>)
                    at fulfilled (E:\vmware\Loop\GitHub\e2e\node_modules\tslib\tslib.js:112:62)
                    at processTicksAndRejections (node:internal/process/task_queues:96:5)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>